cmd/install.go

```diff
@@ -47,38 +53,55 @@ And to see options for a specific app before installing, run:
 		return nil
 	}
 
-	command.AddCommand(apps.MakeInstallOpenFaaS())
-	command.AddCommand(apps.MakeInstallMetricsServer())
-	command.AddCommand(apps.MakeInstallInletsOperator())
-	command.AddCommand(apps.MakeInstallCertManager())
-	command.AddCommand(apps.MakeInstallOpenFaaSIngress())
-	command.AddCommand(apps.MakeInstallNginx())
-	command.AddCommand(apps.MakeInstallChart())
-	command.AddCommand(apps.MakeInstallLinkerd())
-	command.AddCommand(apps.MakeInstallCronConnector())
-	command.AddCommand(apps.MakeInstallKafkaConnector())
-	command.AddCommand(apps.MakeInstallKubeStateMetrics())
-	command.AddCommand(apps.MakeInstallMinio())
-	command.AddCommand(apps.MakeInstallPostgresql())
-	command.AddCommand(apps.MakeInstallKubernetesDashboard())
-	command.AddCommand(apps.MakeInstallIstio())
-	command.AddCommand(apps.MakeInstallCrossplane())
-	command.AddCommand(apps.MakeInstallMongoDB())
-	command.AddCommand(apps.MakeInstallRegistry())
-	command.AddCommand(apps.MakeInstallRegistryIngress())
-	command.AddCommand(apps.MakeInstallTraefik2())
-	command.AddCommand(apps.MakeInstallGrafana())
-	command.AddCommand(apps.MakeInstallArgoCD())
-	command.AddCommand(apps.MakeInstallPortainer())
-	command.AddCommand(apps.MakeInstallTekton())
-	command.AddCommand(apps.MakeInstallJenkins())
-	command.AddCommand(apps.MakeInstallLoki())
-	command.AddCommand(apps.MakeInstallNATSConnector())
-	command.AddCommand(apps.MakeInstallOpenFaaSLoki())
-	command.AddCommand(apps.MakeInstallNfsProvisioner())
-	command.AddCommand(apps.MakeInstallRedis())
+	for _, app := range AppList {
+		command.AddCommand(app.Installer())
+	}
 
 	command.AddCommand(MakeInfo())
 
 	return command
 }
+
+var AppList = map[string]ArkadeApp{
```

Comment:
I'm not keen on this being initialised lazily / at a random time.

What other suggestions do you have for running the code in a deterministic way?

Reply:
Could make a func that returns the app list and use that where we need the lists instead of a var in here?